### DISCUSSION TITLE
General: Best Geiger counter

### DISCUSSION THREAD

**Discussion by J0s3f** [Gissio/radpro#92] [General]
Good Day,
I ordered a FNIRSI GC-01 from AliExpress and received the counterfeit version. Thankfully I got my money back, but before I order another, I'd like to ask your advice on which device is best to use with the firmware.
My use for it will be to play around with it and then probably later connect it to one of my SBCs and share the data online. 
I ordered the GC-01 because it seemed the easiest to flash, but I already have all the hardware to flash other devices, so that's not really an issue. And since there seem to be fakes of the GC-01 and probably not of some other devices, maybe I should just buy another one?

**Comment by jonwhite227** [comment]
I have bought two GC-01 and an S-1 digital multimeter from the **'Fnirsi Official Store'** on AliExpress with no issues or counterfeits. Always a good quick service. Bosean FS-5000 is also a good quality device, and also supported by RadPro.

**Comment by ricsim** [comment]
What do you mean with counterfeit?
It does not get more Chinese then Fnirsi, and I am not aware of counterfeit products. (Some OEMs sell the same device, but it IS manufactured by the same people, just maybe with different color plastics.)

There are 3 MCUs and 3 or 4 different geiger miller tubes, and Fnirsi puts in whatever they have in stock.

Bosean FS-5000 is great if you have the ST Flasher. (they regulate the HV for the tube. With the GC-01 it is fixed and not regulated.)
It seems that Bosean committed that new versions will always use the J321 tube. (which is good, the smaller tubes J613/614 are not good)

**Reply by J0s3f** [reply]
The device with the CACHIP MCU is counterfeit and not manufactured by FNIRSI, at least that's the information I got back from the official FNIRSI email address. Based on this information from FNIRSI, AliExpress gave me my money back.
How to spot such a device is described here: https://github.com/Gissio/radpro/issues/48#issuecomment-2040848370

**Reply by ricsim** [reply]
Anyways, I was lucky as I received one with WCH MCU and one with Geehy MCU.
Geehy is a bit nicert (USB allways on, more Flash for data storage, ...)
But both are OK. I was lucky with the geiger tubes as both had the M4011.

I recently bought a Bosean FS-5000.
Mine came with the J321 (both are OK M4011 is very similar) and it works great.
You just need the ST-Link flasher to flash Gissios FW.
I would recommand the Bosean FS-5000.

**Comment by Montecri** [comment]
> Good Day, I ordered a FNIRSI GC-01 from AliExpress and received the counterfeit version. Thankfully I got my money back, but before I order another, I'd like to ask your advice on which device is best to use with the firmware. My use for it will be to play around with it and then probably later connect it to one of my SBCs and share the data online. I ordered the GC-01 because it seemed the easiest to flash, but I already have all the hardware to flash other devices, so that's not really an issue. And since there seem to be fakes of the GC-01 and probably not of some other devices, maybe I should just buy another one?

Hi,

It appears that FS-5000 is technically better (closed loop HV, more storage, etc); unfortunately, mine came with the tiny tube and I had to replace it (now they commit to M4011/J321, it appears).

However, FNIRSI has an official store on Aliexpress, so there's no way to go wrong purchasing from there.

What I find superior on FNIRSI is the LCD, better viewing angles and contrast, IMO. Also, more buttons. I like buttons :)

Regards,

- Cristiano

**Reply by jonwhite227** [reply]
I have used the Fnirsi Official Store on AliExpress several times with no issues.

**Reply by Montecri** [reply]
> I have used the Fnirsi Official Store on AliExpress several times with no issues.

Same.

**Reply by J0s3f** [reply]
I now ordered a FS-5000 from their official store where they guarantee to provide one with J321. I hope for the best.

**Reply by DG0MG** [reply]
> closed loop HV

Cristiano, where did you get this information? Can i confirm this somewhere?
That would be a great step for BOSEAN!

regards, Steffen

**Comment by JTknives** [comment]
I love the fs-5000. I currently mod them with 11A tubes and resell them. 

![image](https://github.com/Gissio/radpro/assets/171413119/7be04915-fbec-4943-8814-6611effc589d)

![image](https://github.com/Gissio/radpro/assets/171413119/ebc0b54a-4ac9-4e0f-a02b-1bb00d57c231)

![image](https://github.com/Gissio/radpro/assets/171413119/05688ada-7805-453e-bf29-fb0870a4e698)

![image](https://github.com/Gissio/radpro/assets/171413119/95b7136a-5653-4d87-94a3-0c7ca6f81102)

**Reply by DG0MG** [reply]
> I love the fs-5000. I currently mod them with 11A tubes and resell them.

You should try to opening the SBT-11A and using it without the Bakelite case, that could be smaller: [https://www.geigerzaehlerforum.de/index.php?topic=656.0](https://www.geigerzaehlerforum.de/index.php?topic=656.0)
